app/views/tasks/new.html.erb

```diff
@@ -1,6 +1,7 @@
 <h1>Add a New Task</h1>
 <form method="POST" action="/tasks" enctype="multipart/form-data">
+  <input type="hidden" name="authenticity_token" value="<%= Rack::Protection::AuthenticityToken.token(env['rack.session']) %>">
   <label for="description">Task Description</label>
   <input type="text" name="description" id="description">
-  <input type="submit" value="Save">
+  <button type="submit">Save</button>
```

Comment:
This is another change that shouldn't have been necessary that I'm curious about.